cms/templates/maintenance/_announcement_edit.html

```diff
@@ -0,0 +1,47 @@
+<%page expression_filter="h"/>
+<%inherit file="base.html" />
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML, Text
+%>
+<%block name="title">${_('Edit Announcement')}</%block>
+<%block name="viewtitle">
+<h3 class="info-course">
+    <span>${_('Edit Announcement')}</span>
+</h3>
+</%block>
+
+<%block name="viewcontent">
+<section class="container maintenance-content">
+    <div class="container studio-view maintenance-form">
+        <form class="form-create" method="post" action="${action_url}">
+            <div class="wrapper-form announcement-container">
+                <input type="hidden" name="csrfmiddlewaretoken" value="${csrf_token}"/>
+                <div class="announcement-item">
+                  ${form.as_p() | n} ## xss-lint: disable=mako-invalid-html-filter
```

Comment:
nit: this `xss-lint` comment is rendering to the HTML:

![xss-comment](https://user-images.githubusercontent.com/7556571/51825196-2a82d680-2334-11e9-9371-88af60ed25e2.png)

Reply:
Ah, those have to be on their own line evidently. Thanks for catching that.